dapps/evancore.vue.libs/src/components/table/table.vue

```diff
@@ -45,12 +46,25 @@ export default Component;
 @import '~@evan.network/ui/src/style/utils';
 
 .evan-table-wrapper {
-  margin: 0;
   width: 100%;
 
+  &.show-scrollbar {
+    overflow-y: scroll;
+    // This is needed for webkit browsers
+    // https://stackoverflow.com/a/31278448
+    &::-webkit-scrollbar {
+      -webkit-appearance: none;
+      width: 7px;
+    }
+    &::-webkit-scrollbar-thumb {
+      border-radius: 4px;
+      background-color: rgba(0, 0, 0, 0.5);
+      -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
```

Comment:
vendor prefix will be added by scss I think

```suggestion
      box-shadow: 0 0 1px rgba(255, 255, 255, 0.5);
```